### ISSUE TITLE
On learning rate range test

### ISSUE THREAD

**Issue by deepaksuresh** [bckenstler/CLR#5]
Hi @bckenstler ,
Thanks a lot for sharing your implementation. I just read the paper on [cyclical learning rate](https://arxiv.org/abs/1506.01186). I'd like to know how you dealt with the following 
1. How do you choose the number of epochs to run the model?
2. When I run the LR range test, and estimate accuracy on validation set, I get an accuracy of 0.5 for almost all the learning rates. Since, the model is hardly trained, it is behaving as a random classifier. Then, the validation accuracy around 0.5 looks justified. But, this is not what the accuracy vs learning curve look like in the paper. How did you deal with this?
Thanks

**Comment by zchrissirhcz** [comment]
@deepaksuresh I've tried to reproduce the CLR paper's experiment on CIFAR-10, especially of Figure 3's first two rows and Table1. However, what I got is different from the paper. If interested, you can try my code https://github.com/zchrissirhcz/Cyclical-Learing-Rates
